stateful-datatable-test/src_test/com/axonivy/demo/statefuldatatable/test/StatefulDatatableDemoWebIT.java

```diff
@@ -28,7 +28,7 @@
  * 3_ start "showDatatableRepo" process
  * 4_ verify that the table of "Products" is displayed.
  * 5_ add a New "Product" into the business repo.
- * 
+ *
  */
 @IvyTest
 @IvyWebTest(headless = true)
```

Comment:
remove headless :D

Reply:
didn't invent it :sweat_smile:  ... but I think it ok and up to the developer prefs, as long as headless evaluates to true